types/types.go

```diff
@@ -303,7 +303,7 @@ type BuildConfig struct {
 	CacheFrom          StringList            `mapstructure:"cache_from" yaml:"cache_from,omitempty" json:"cache_from,omitempty"`
 	CacheTo            StringList            `mapstructure:"cache_to" yaml:"cache_to,omitempty" json:"cache_to,omitempty"`
 	NoCache            bool                  `mapstructure:"no_cache" yaml:"no_cache,omitempty" json:"no_cache,omitempty"`
-	AdditionalContexts MappingWithEquals     `mapstructure:"additional_contexts" yaml:"additional_contexts,omitempty" json:"additional_contexts,omitempty"`
+	AdditionalContexts Mapping               `mapstructure:"additional_contexts" yaml:"additional_contexts,omitempty" json:"additional_contexts,omitempty"`
```

Comment:
Spec says `additional_contexts` can be sequence as well: https://github.com/compose-spec/compose-spec/blob/f5002178151d5b15f84da0deecb66a287afde19c/build.md#additional_contexts

Reply:
indeed. but still it's not valid to provide a key without a value, so this isn't a MappingWithEquals.
I'll add support for converting a string list into a `Mapping`